From d75bc3b69151a9d0391309e6bb51784f3b6b9a83 Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Mon, 16 Dec 2013 09:26:03 +0100
Subject: [PATCH] db_flatstore: update timestamp for kamailio specific rpc
 rotate

- do also in 'rotate' command, not only in 'k_rotate'
- from a patch by Abdelkader Allam

(cherry picked from commit b15aefa31e45f1ad6a45b64905f9fae24e0f790e)
---
 modules/db_flatstore/flat_rpc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules/db_flatstore/flat_rpc.c b/modules/db_flatstore/flat_rpc.c
index 2fe0d0f..5145d83 100644
--- a/modules/db_flatstore/flat_rpc.c
+++ b/modules/db_flatstore/flat_rpc.c
@@ -32,6 +32,7 @@
 
 #include "flat_rpc.h"
 #include "flatstore_mod.h"
+#include "km_flatstore_mod.h"
 
 #include <time.h>
 
@@ -43,6 +44,7 @@
  */
 static void rotate(rpc_t* rpc, void* c)
 {
+	*km_flat_rotate = time(0);
 	*flat_rotate = time(0);
 }
 
-- 
1.8.5.2

